t/validator.t

```diff
@@ -66,6 +55,31 @@ subtest 'Everything but NoWarnings' => sub {
         ok !tainted( untaint_engine_type( taint( 'SQLite' ) ) ), 'launder taint';
     };
 
+    subtest 'untaint_ip_address' => sub {
+        is scalar untaint_ip_address( '192.0.2.1' ),                              '192.0.2.1',                              'accept: 192.0.2.1';
+        is scalar untaint_ip_address( '192.0.2' ),                                undef,                                    'reject: 192.0.2';
+        is scalar untaint_ip_address( '192.0.2.1:3306' ),                         undef,                                    'reject: 192.0.2.1:3306';
+        is scalar untaint_ip_address( '2001:db8::' ),                             '2001:db8::',                             'accept: 2001:db8::';
+        is scalar untaint_ip_address( '2001:db8::/32' ),                          undef,                                    'reject: 2001:db8::/32';
+        is scalar untaint_ip_address( '2001:db8:ffff:ffff:ffff:ffff:ffff:ffff' ), '2001:db8:ffff:ffff:ffff:ffff:ffff:ffff', 'accept: 2001:db8:ffff:ffff:ffff:ffff:ffff:ffff';
+        is scalar untaint_ip_address( '2001:db8:ffff:ffff:ffff:ffff:ffff' ),      undef,                                    'reject: 2001:db8:ffff:ffff:ffff:ffff:ffff';
+        is scalar untaint_ip_address( '2001:db8::255.255.255.254' ),              '2001:db8::255.255.255.254',              'accept: 2001:db8::255.255.255.254';
+        is scalar untaint_ip_address( '2001:db8::255.255.255' ),                  undef,                                    'reject: 2001:db8::255.255.255';
+        is scalar untaint_ip_address( '::1' ),                                    '::1',                                    'accept: ::1';
```

Comment:
Could you test with `:::1` (triple colons), I think this should fail, and it would be nice to have it tested, so we have a test case where the IPv6 regex succeed but the `Zonemaster::Engine::Net::IP::ip_is_ipv6()` fails.

Reply:
Done.